.github/workflows/release.yml

```
@@ -19,6 +18,9 @@
       - name: Checkout code
         uses: actions/checkout@v3
 
+      - name: Gradle Wrapper Validation
+        uses: gradle/wrapper-validation-action@v1
```

actionlint (1.7.4)

the runner of "gradle/wrapper-validation-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)